model/task/db.go

```diff
@@ -2880,6 +2882,8 @@ func abortAndMarkResetTasks(ctx context.Context, filter bson.M, taskIDs []string
 			AbortedKey:           true,
 			AbortInfoKey:         AbortInfo{User: caller},
 			ResetWhenFinishedKey: true,
+			// this function should override the automatic restart key if it is set
+			IsAutomaticRestartKey: false,
```

Comment:
todo: what's this for?

Reply:
This was to ensure that clicking "restart version & abort in progress tasks" still gets respected if some of the tasks were marked for auto restart, but hadn't completed yet.  

This isn't necessary anymore given the changes I made to the `SetAborted` DB update so I'm removing this